API/Parser/Parser.cs

```diff
@@ -67,40 +67,39 @@ public static class Parser
         private static readonly Regex SpecialTokenRegex = new Regex(@"SP\d+",
             MatchOptions, RegexTimeout);
 
+        private const string SeriesExpr = $@"(?<Series>{Balanced}?)";
+        private const string VolumeExpr = @"(?<Volume>\d+(\.\d)?(-\d+(\.\d)?)?)";
+        private const string ChapterExpr = @"(?<Chapter>\d+(\.\d)?(-\d+(\.\d)?)?)";
+        private const string BalancedParen = @"(?:[^()]|(?<open>\()|(?<-open>\)))*?(?(open)(?!))";
+        // non greedy matching of a string where parenthesis and brackets are balanced
+        private const string Balanced = @"(?:[^()\[\]]|(?<open>\()|(?<-open>\))|(?<openbracket>\[)|(?<-openbracket>\]))*?(?(openbracket)(?!))(?(open)(?!))";
+        // French Volume: tome, t, t. -> volume
+        private const string VolumePrefixes = @"(v|t|(v\.|vol\.?|volume|t\.|tome)\s?)";
+        // n°,n. -> numero sign
+        private const string chapterPrefixes = @"(c|chapter[-\s]+|issue\s?#?|#|(n°|n\.|Ep\.?)[-\s]*)";
```

Comment:
I think it would be beneficial to have unit tests for these, so when adding a new case, it can be easily tested without worrying about breakage.

Reply:
They are strings that are reused in regex. This is meant to ensure that regex can reuse the same format for a chapter, etc.  Just like the regex themselves get tested with the parsing functions, these pieces of strings get tested with them.